core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -274,6 +274,13 @@ public enum Property {
       PropertyType.BOOLEAN, "Enables JVM metrics functionality using Micrometer", "2.1.0"),
   GENERAL_MICROMETER_FACTORY("general.micrometer.factory", "", PropertyType.CLASSNAME,
       "Name of class that implements MeterRegistryFactory", "2.1.0"),
+  GENERAL_LOCATION_MODE("general.location.mode", "locality", PropertyType.LAST_LOCATION_MODE,
```

Comment:
@ctubbsii Should this be a general property or a tserver property?

Reply:
So it appears that a general property cannot be set in zookeeper.  I am leaning towards changing this to a tserver property unless you have a reason why not to.

Reply:
You're right. tserver would make more sense